assets/js/modules/analytics/components/settings/SettingsEdit.test.js

```diff
@@ -61,8 +61,11 @@ describe( 'SettingsEdit', () => {
 		};
 		const { registry } = render( <SettingsEdit />, { setupRegistry } );
 
-		expect( registry.select( STORE_NAME ).getAccountID() ).toBe( existingTag.accountID );
-		expect( registry.select( STORE_NAME ).getPropertyID() ).toBe( existingTag.propertyID );
-		expect( registry.select( STORE_NAME ).hasErrors() ).toBeFalsy();
+		waitFor( () => {
```

Comment:
Missing `await` – also, let's not put all the assertions into `waitFor`. We should only wait until the datastore state has stopped changing. It should be sufficient to have everything received ahead of time like it is now, so if that's still causing problems, we'll have to see what is changing in the store for it to complain about.